jwst/datamodels/container.py

```diff
@@ -348,12 +347,6 @@ def _assign_group_ids(self):
                                              ''.join(params[3:6]), params[6]]))
                 model.meta.group_id = group_id
             except TypeError:
-                params_dict = dict(zip(unique_exposure_parameters, params))
```

Comment:
What was the issue here?  Not that I'd miss this code, just curious.

Reply:
Yeah, this is a very unnecessary warning that applies to all data that isn't NIRCam.  So you are cursed to get this warning if you're not using NIRCam data.  Not a good warning.